src/containers/availableTeams/index.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react';
+import LeaderboardSpace, {
+  TabItem,
+} from '../../components/leaderboard/leaderboardSpace';
+import { Pagination, Divider } from 'antd';
+import PageHeader from '../../components/pageHeader';
+import styled from 'styled-components';
+import { Button } from 'antd';
+import { ArrowRightOutlined } from '@ant-design/icons';
+
+const ContentContainer = styled.div`
+  padding: 100px 134px;
+`;
+
+const TeamsContainer = styled.div`
+  margin: 80px 0px 40px;
+`;
+
+const TeamPagination = styled(Pagination)`
+  display: flex;
+  justify-content: flex-end;
+`;
+
+const TeamDivider = styled(Divider)`
+  margin-top: 10px;
+  margin-bottom: 20px;
+`;
+
+const ArrowLink = styled(ArrowRightOutlined)`
+  line-height: 0px;
+  font-size: 20px;
+`;
+
+const dummy: TabItem[] = [
```

Comment:
Can you make sure there's a ticket for connecting this to the backend?

Reply:
Part of the connect all team routes ticket